.lando.yml

```diff
@@ -1,10 +1,14 @@
 name: sous-project
-recipe: drupal8
+recipe: drupal9
 config:
   webroot: web
-  drush: 9
-  php: '7.2'
+  php: '7.3'
+services:
+  appserver:
+    run:
```

Comment:
This will fire the composer install on `lando start` so I'm not sure we should have this in here. We have the build scripts to handle this and I'm thinking we should just keep it there and not double up on the install.

Reply:
@americkson Subsequent composer installs would run fast if it was already run once before.  I would like to leave it here as it makes the out of the box / testing experience better (the site will redirect to the install page after running just the `lando start` command).

Reply:
That makes sense and will provide better install process.